server/list.php

```diff
@@ -0,0 +1,13 @@
+{
+	"data": [{
+		"title": "Get eggs",
+		"description": "Go to the store and ...",
```

Comment:
why do we have the ...?

Reply:
It's going to be a truncated version of the description, the server will truncate it before we get it.